modules/profilereview/src/Auth/Process/ProfileReview.php

```diff
@@ -22,8 +22,8 @@ class ProfileReview extends ProcessingFilter
     const STAGE_SENT_TO_NAG = 'profilereview:sent_to_nag';
 
     const REVIEW_PAGE = 'review.php';
```

Comment:
Should this be just `review`, or does it still need the `.php` extension on it?

Reply:
It needs the .php extension still. The "old" way was confusing because there were two php files: the template and the controller. The "new" way uses a twig template and a PHP Class for the controller. I may or may not convert to the controller class. Until then, the route will still end in .php